src/main/java/org/ambraproject/wombat/controller/PowerPointController.java

```diff
@@ -119,7 +120,7 @@ public void download(HttpServletRequest request, HttpServletResponse response,
           String figureDoi = (String) figure.get("doi");
           return figureDoi.equals(assetPointer.getAssetDoi());
         })
-        .findAny();
+        .collect(MoreCollectors.toOptional());
```

Comment:
Similar to `MoreCollectors.onlyElement()`, `toOptional()` expects 0 or 1 elements but, unlike `findAny()`, will throw an exception if there is more than 1.